dev_utils/README.md

```diff
@@ -34,10 +34,16 @@ Run the go proxy from the root directory
 
 ```bash
 export GO111MODULE=on
+export SERVER_CONFFILE=dev_utils/config.yaml
 go build main.go
 ./main
 ```
 
+## Test with s3 configuration file
+To test the implementation locally with a file exported from the 
+[login](https://login.ega.nbis.se), change the `host_base` and
+`host_bucket` to `localhost:8000`, where the S3 Upload Proxy is running.
+
```

Comment:
This needs to be more generic since it is others than us that will use this one.

Reply:
Good comment, I will update it

Reply:
@jbygdell should I leave the link to our login as an example, just to make it easier for us to test or does it make sense to have something general, such as `To test the implementation locally with a configuration file for the s3cmd`

Reply:
There is an s3config file in the dev-tools folder.
If you want to reference the actual login portal link it to the sda-pipeline repo, as it is there it can be found.